content/doc/upgrade-guide/2.7.adoc

```diff
@@ -0,0 +1,83 @@
+---
+layout: simplepage
+title:  Jenkins Upgrade Guide
+section: doc
+---
+
+== Upgrading to Jenkins LTS 2.7.x
+
+Each section covers the upgrade from the previous LTS release, the section on 2.7.1 covers the upgrade from 1.651.3.
+
+=== Upgrading to Jenkins LTS 2.7.4
+
+This release only fixed a regression introduced in 2.7.3. No upgrade notes for this release exist.
+
+=== Upgrading to Jenkins LTS 2.7.3
+
+==== Enforced upgrade of detached plugins
+
+https://issues.jenkins-ci.org/browse/JENKINS-37041[JENKINS-37041]
+
+Jenkins will now ensure that detached plugins, i.e. those whose functionality used to be part of Jenkins core but was moved into a plugin, are always at least the bundled version. A message will be printed to the log informing administrators of this:
+
+ Upgraded detached plugins (and dependencies)
+
+=== Upgrading to Jenkins LTS 2.7.2
+
+==== Support for HTTP_PROXY environment variable in remoting component
+
+https://issues.jenkins-ci.org/browse/JENKINS-32326[JENKINS-32326]
+
+Support for HTTP proxy exclusion in the build agent launcher was added. Build agents that define this variable, but whose connection to the Jenkins master relies on it not being enabled may now fail to to connect to Jenkins. The solution is to fix the proxy configuration as passed to the agent process.
+
+=== Upgrading to Jenkins LTS 2.7.1
```

Comment:
I would mention that JNLP3 may be randomly enabled on 2.7.1 and 2.7.2 and it may cause SOME communication issues. Protocol disabling System property can be used as a workaround